test/SAMtags/README.md

```diff
@@ -0,0 +1,35 @@
+MM and ML auxiliary tags
+========================
+
+The purpose of these test files is to test parsing of the MM and ML
+tags.  These succint MM and ML tags are present in the .sam files,
+with a more human readable expanded form in the .txt files.
+Developers should check whether their implementation is able to
+convert between the two forms.
+
+The .sam files are SAM format, but the only fields used for this
+test are the reverse-complementation flag (FLAG bit 0x10), the
+sequence, and the MM and ML tags.
+
+The .txt files uses one line per base, with a blank line separating
+sequeces.  Each line consists of two table separated fields
+representing the top (original as-sequenced orientation) and bottom
+strand.
+
+Each field in the .txt files is a concatenation of the canonical base
+and any modifications with their associated probability, expressed
+here as a rounded-down percentage.  The modification and percentage
+are joined together with no punctuation, but for ChEBI numeric codes
```

Comment:
wouldn't it be simpler to write a test for this if there was a clear separator between the modification and the percent?

Reply:
There is - it's MM-chebi.sam/txt.

It's also included in the README as documentation for the format of the .txt file.

Reply:
Ah, did you mean wouldn't it be easier if we had a different format in the .txt file than the one chosen?

Ifo so then maybe, but I wanted something that is trivially human readable and importantly in a different mod / qual ordering than the input data to test that people have interpretted how to decode them properly.  I'm open to other ideas.